### ISSUE TITLE
Hide traditional or simplified example sentences for Chinese

### ISSUE THREAD

**Issue by shiki-tm** [yomidevs/wiktionary-to-yomitan#167]
hi boss im using kty zh-en dict and i love the example sentences but i dont need to see the simplified characters version so i want to hide it with css. but it looks like theyre called the same when i inspect element, right? Both lang are called ja. 
![image-58](https://github.com/user-attachments/assets/88618bc9-a281-4d6e-afc7-9580b966cf5b)
![image-53](https://github.com/user-attachments/assets/f3109c44-47c3-43a7-9d3e-33d9d3b91427)
is there a way to just hide the 2nd one ? but theres also sometimes more than 1 meaning with more sentences. thanks

**Comment by shiki-tm** [comment]
Thank this worked! so what happened to your other comment about the font being wrong? did they fix it or should i force the font with css?

**Comment by daxida** [comment]
> My PC language is zh-TW, but it still shows the characters in JP (because they are marked as so by the devs of kaikki-to-yomitan I guess?)

It's the Yomitan default. `wty` doesn't populate the lang attribute so it falls back to that. It would just change to zh, since wiktextract doesn't give us more details. This could be fixed.

---

As for the simp/trad, take 惹:

[wiktionary](https://en.wiktionary.org/wiki/%E6%83%B9)
https://kaikki.org/dictionary/Chinese/meaning/%E6%83%B9/%E6%83%B9/%E6%83%B9.html

---

<img width="1500" height="800" alt="Image" src="https://github.com/user-attachments/assets/7c4e4f39-287b-4160-a934-5a23a1206e59" />

---

There is no way to tell which one is simp/trad by looking at the data.

Wiktionary though seems to automatically default to simp first. You can see how the template expands to simp/trad.

<img width="1200" height="300" alt="Image" src="https://github.com/user-attachments/assets/e7cc6271-a69a-44e7-8964-7ab0bee047dd" />

--- 

Do you have any examples of trad going first? (please don't post pictures, just the word so I copy paste it)

**Comment by username-011** [comment]
> Wiktionary though seems to automatically default to simp first. 

No, on the screenshot you showed, it is Traditional as a default. Then (since the original data doesn't have the simplified variant) I guess it is being automatically converted to simplified.

> Do you have any examples of trad going first?

First exactly in your screenshot directly above this post. Second in the same word, 惹:
<img width="1241" height="908" alt="Image" src="https://github.com/user-attachments/assets/f589e5d9-8486-463c-8c93-9d00714ab196" />
Both the page and the data for this goes in the order: `Trad, simp; simp, trad; simp, trad`. 
So trad-simp are not in a fixed order. I think they go in the order of which one is the original source, and then the converted one.

> There is no way to tell which one is simp/trad by looking at the data.  ...since wiktextract doesn't give us more details

Yes, I guess it would require an improvement in the data by wiktextract.

zh-x/zh-q default to traditional by default, needs `simp=y` to be simplified by default. Relevant wiki article:
https://en.wiktionary.org/wiki/Template:zh-x

**Comment by daxida** [comment]
> First exactly in your screenshot directly above this post

Haha, what a mistake. Well, I'll let them know in case they want to add that information, otherwise any fix we would do here won't be much different than your

```
:is([data-dictionary="kty-zh-en"], [data-dictionary="kty-zh-zh"]) [data-sc-content="details-entry-examples"] [data-sc-content="extra-info"]:nth-child(odd) {
  display: none;
}
```

**Comment by daxida** [comment]
See https://yomidevs.github.io/wiktionary-to-yomitan/css/#examples-chinese

It will work once I make a new release >=09/19.

Edit: new release is up.
